packages/paste-website/src/pages/blog/index.mdx

```diff
@@ -4,6 +4,8 @@ export const meta = {
   slug: '/blog/',
 };
 
+import {PageHeader, PageHeaderDetails, PageHeaderHeading} from '@twilio-paste/page-header';
+
```

Comment:
suggestion(non-blocking): unsure on feasibility, but... similarly to what I did for Component pages, which you haven't had to update the mdx files for, if we were going to make a big improvement, we could explore layouts for pattern pages, foundation pages, content pages etc, use the page `meta` object to pass in the data needed to the layout to render the page header for each. 

then you wouldn't need to import any of the page header compoents into the mdx pages.

Reply:
yeah that would be ideal...